DOC-1226 overview style edits (#257)
redpanda-data/cloud-docs: modules/get-started/pages/cloud-overview.adoc

== Redpanda Cloud cluster types

> Redpanda offers three types of fully-managed cloud clusters. All products have access to unlimited storage and 280+ data connectors with xref:develop:connect/about.adoc[Redpanda Connect].

Reviewer: ```suggestion
Redpanda offers fully-managed cloud clusters for Serverless, BYOC, and Dedicated. All products have access to unlimited storage and 280+ data connectors with xref:develop:connect/about.adoc[Redpanda Connect].
```
